### PR TITLE
feat: add list of roles and members

### PR DESCRIPTION
**Pull request by ematipico**

### CODE REVIEW

File: MEMBERS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Current members
+
+Members are listed in alphabetical order, by name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
+
+## Steward
+
+
+## Core
+
```
Comment (sarah11918) [inline MEMBERS.md:9]:
```suggestion

- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918]
```

**Review by sarah11918 — COMMENTED** [review]
There's my name! But also, maybe it's easier to read if only the GitHub handle is linked? Whatever we end up deciding! I added mine like that for comparison.

File: MEMBERS.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
+- [Emanuele Stoppa @ematipico](https://github.com/ematipico)
```
Comment (ematipico) [inline MEMBERS.md:11]:
```suggestion
- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
```

File: MEMBERS.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Current members
+
+Members are listed in alphabetical order, by name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
```
Comment (ematipico) [inline MEMBERS.md:3]:
```suggestion
Members are listed in alphabetical order, by first name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
```

File: MEMBERS.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+Members are listed in alphabetical order, by first name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
+
+## Steward
+
+
+## Core
+
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
+- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
```
Comment (Princesseuh) [inline MEMBERS.md:11]:
```suggestion
- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
```

File: MEMBERS.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+## Steward
+
+
+## Core
+
+- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
+- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
+
```
Comment (yanthomasdev) [inline MEMBERS.md:13]:
```suggestion
- Yan Thomas [@yanthomasdev](https://github.com/yanthomasdev)

```

Comment (TheOtterlord) [inline MEMBERS.md:13]:
```suggestion
- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
- Reuben Tier [@TheOtterlord](https://github.com/TheOtterlord)

```

File: MEMBERS.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+Members are listed in alphabetical order, by first name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
+
+## Steward
+
+
+## Core
+
+- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
+- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
```
Comment (bholmesdev) [inline MEMBERS.md:12]:
```suggestion
- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
- Ben Holmes [@bholmesdev](https://github.com/bholmesdev)
```

File: MEMBERS.md (hunk 5 above)
Comment (ascorbic) [inline MEMBERS.md:13]:
```suggestion
- Matt Kane [@ascorbic](https://github.com/ascorbic)
```

File: MEMBERS.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+Members are listed in alphabetical order, by first name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
+
+## Steward
+
+
+## Core
+
+- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
+- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
```
Comment (florian-lefebvre) [inline MEMBERS.md:11]:
```suggestion
- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
- Florian Lefebvre [@florian-lefebvre](https://github.com/florian-lefebvre)
```

File: MEMBERS.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
+- Yan Thomas [@yanthomasdev](https://github.com/yanthomasdev)
+## Maintainers
```
Comment (delucis) [inline MEMBERS.md:14]:
```suggestion

## Maintainers
```

File: MEMBERS.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+## Steward
+
+
+## Core
+
+- Ben Holmes [@bholmesdev](https://github.com/bholmesdev)
+- Chris Swithinbank [@delucis](https://github.com/delucis)
+- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
+- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
+- Florian Lefebvre [@florian-lefebvre](https://github.com/florian-lefebvre)
```
Comment (HiDeoo) [inline MEMBERS.md:14]:
```suggestion
- Florian Lefebvre [@florian-lefebvre](https://github.com/florian-lefebvre)
- HiDeoo [@hideoo](https://github.com/hideoo)
```

File: MEMBERS.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+Members are listed in alphabetical order, by first name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
+
+## Steward
+
+
+## Core
+
+- Ben Holmes [@bholmesdev](https://github.com/bholmesdev)
+- Chris Swithinbank [@delucis](https://github.com/delucis)
```
Comment (itsMapleLeaf) [inline MEMBERS.md:11]:
```suggestion
- Chris Swithinbank [@delucis](https://github.com/delucis)
- Darius [@itsMapleLeaf](https://github.com/itsMapleLeaf)
```

File: MEMBERS.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+- Darius [@itsMapleLeaf](https://github.com/itsMapleLeaf)
+- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
+- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
+- Florian Lefebvre [@florian-lefebvre](https://github.com/florian-lefebvre)
+- HiDeoo [@hideoo](https://github.com/hideoo)
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
+- Reuben Tier [@TheOtterlord](https://github.com/TheOtterlord)
+- Yan Thomas [@yanthomasdev](https://github.com/yanthomasdev)
+
```
Comment (aFuzzyBear) [inline MEMBERS.md:20]:
```suggestion
- Fuzzy [@aFuzzyBear](https://github.com/afuzzybear
```

File: MEMBERS.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+- Chris Swithinbank [@delucis](https://github.com/delucis)
+- Darius [@itsMapleLeaf](https://github.com/itsMapleLeaf)
+- Emanuele Stoppa [@ematipico](https://github.com/ematipico)
+- Erika "Princesseuh" [@Princesseuh](https://github.com/princesseuh)
+- Florian Lefebvre [@florian-lefebvre](https://github.com/florian-lefebvre)
+- HiDeoo [@hideoo](https://github.com/hideoo)
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
+- Reuben Tier [@TheOtterlord](https://github.com/TheOtterlord)
+- Yan Thomas [@yanthomasdev](https://github.com/yanthomasdev)
+- Fuzzy [@aFuzzyBear](https://github.com/afuzzybear
```
Comment (matthewp) [inline MEMBERS.md:20]: Matthew Phillips [@matthewp](https://github.com/matthewp)

Comment (matthewp) [inline MEMBERS.md:20]:
```suggestion
- Fuzzy [@aFuzzyBear](https://github.com/afuzzybear)
```

Comment (matthewp) [inline MEMBERS.md:20]:
```suggestion
- Matthew Phillips [@matthewp](https://github.com/matthewp)
```

Comment (matthewp) [inline MEMBERS.md:20]:
```suggestion
- Fuzzy [@aFuzzyBear](https://github.com/afuzzybear
- Matthew Phillips [@matthewp](https://github.com/matthewp)
```

File: MEMBERS.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+- Sarah Rainsberger [@sarah11918](https://github.com/sarah11918)
+- Reuben Tier [@TheOtterlord](https://github.com/TheOtterlord)
+- Yan Thomas [@yanthomasdev](https://github.com/yanthomasdev)
+- Fuzzy [@aFuzzyBear](https://github.com/afuzzybear)
+- Matthew Phillips [@matthewp](https://github.com/matthewp)
+## Maintainers
+
+## Core residency
+
+- Matt Kane [@ascorbic](https://github.com/ascorbic)
```
Comment (itsMapleLeaf) [inline MEMBERS.md:27]:
```suggestion
- Darius [@itsMapleLeaf](https://github.com/itsMapleLeaf)
- Matt Kane [@ascorbic](https://github.com/ascorbic)
```

File: MEMBERS.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+Members are listed in alphabetical order, by first name. Members are free to use the full name, GitHub handle, or any other nickname they wish to be addressed.
+
+## Steward
+
+
+## Core
+
+- Ben Holmes [@bholmesdev](https://github.com/bholmesdev)
+- Chris Swithinbank [@delucis](https://github.com/delucis)
+- Darius [@itsMapleLeaf](https://github.com/itsMapleLeaf)
```
Comment (itsMapleLeaf) [inline MEMBERS.md:12]:
```suggestion
```
moved to residency section

**Pull request merged**